content/en/docs/ops/common-problems/_index.md

```diff
@@ -0,0 +1,10 @@
+---
+title: Common Problems
+description: Describes how to identify and resolve common problems in Istio.
+weight: 10
+keywords: [ops]
+aliases:
+    - /help/ops/troubleshooting
```

Comment:
A bit strange that the troubleshooting link is redirected to common problems rather than to the troubleshooting page.